packages/aws-cdk/lib/commands/init/init.ts

```diff
@@ -524,6 +524,6 @@ async function loadInitVersions(): Promise<Versions> {
  * These have been built into the CLI at build time.
  */
 export async function currentlyRecommendedAwsCdkLibFlags() {
-  const recommendedFlagsFile = path.join(__dirname, './init-templates/.recommended-feature-flags.json');
+  const recommendedFlagsFile = path.join(cliRootDir(), 'lib', 'init-templates', '.recommended-feature-flags.json');
```

Comment:
This was incorrect before and would have broken the toolkit-lib and cli-lib-alpha and integ-runner and cdk-cli-wrapper if they were supporting init command.